src/main/flight/gps_rescue.c

```diff
@@ -792,10 +811,13 @@ void gpsRescueUpdate(void)
 
         rescueState.intent.rollAngleLimitDeg = 0.5f * rescueState.intent.velocityItermRelax * gpsRescueConfig()->maxRescueAngle;
         // gradually gain roll capability to max of half of max pitch angle
-
+ 
         if (newGPSData) {
+            rescueState.intent.pitchAngleLimitDeg = gpsRescueConfig()->maxRescueAngle * rescueState.sensor.groundspeedErrorRatioInv;
```

Comment:
Better to use `else` branch (set `rescueState` only in one place)

Reply:
I'm not sure how to avoid this... in many places in the rescue code, these rescueState values are changed sequentially as we pass through the stages of the rescue, and used elsewhere. They are usually only changed at one time, in one place, sequentially, and this is no exception.